### ISSUE TITLE
[FEATURE REQUEST] ANY-USER trict local installation for non-administrative users

### ISSUE THREAD

**Issue by NullPlane** [marticliment/WingetUI#2059]
### Please confirm these before moving forward

- [X] I have searched for my feature proposal and have not found a work-in-progress/duplicate/resolved/discarded issue.
- [X] This proposal is a completely new feature. If you want to suggest an improvement or an enhancement, please use [this template](https://github.com/marticliment/WingetUI/issues/new?assignees=marticliment&labels=enhancement&projects=&template=enhancement-improvement.yml&title=%5BENHANCEMENT%5D+%28Enter+your+description+here%29)

### Describe the new feature

A preset that may be selected under settings or in the First-Start-Wizard,
which allows for purely local installation of packages that does not require
administrative privileges (which may potentially be configured as an option
in settings to be hidden). Therefore for many packages that do not support
local installation a respective script may be generated.

I do not know how well this would integrate with the app but I think it'd be 
a useful feature.

### Describe how this new feature could help users.

Allows to give restricted users the possibility to install applications on their
system which they may need in order to use them but does not require
the system's administrator to give privileges.

Useful for smaller/private organizations or parental child control in case of
a Windows-based supervised system.

**Comment by marticliment** [comment]
Maybe some kind of default installation options could work

**Comment by NullPlane** [comment]
Yes indeed. I do not know of all the possible ways of accomplishing this in Windows and how it will end up influencing the respective apps, especially for apps which do need privileges, but a bypass with `__COMPAT_LAYER=RunAsInvoker` or something, with some checkbox, should be enough for the first time as long as one may set it as a default option.

What's important tho is a setting which allows for blocking the user from changing this setting or hide it from them. Then one will be able to allow anyone on the machine to install software locally and with some drawbacks of course. 

A suitable default installation folder for these apps would be an interesting thought as I am never a fan of AppData but it still is probably the most important folder in this domain. Still a unified path would be great.

**Comment by marticliment** [comment]
Then I am going to rename this feature request to Default Install Options

**Comment by manchbox** [comment]
I am having issues with a few of my package managers/install locations/cli arguments which if we could manually choose directory for packages that don't exist that would be handy.
- PowerShell 7.5 install, the program keeps corrupting, losing install directory, I think perhaps Enviroment Variable issues? Or perhaps something else broken in the background.
- Dotnet package manager
Notice ".NET Tools is enabled and ready to go ver 9.0.201"
CL Argument "C:\Program Files\dotnet\dotnet.exetool"
This is a diffrent issue, could be solved by the editing command line, I obviously need to put a space between .exe and tool. Also Is there anyway the Command argument can be run without the .exe
